### DISCUSSION TITLE
convert column type

### DISCUSSION THREAD

**Discussion by Thornzee** [Budibase/budibase#11117] [Help] [answered]
I am trying to convert column type from text to number but it won't show as an option. any idea why?

![CaptureCOLUMN](https://github.com/Budibase/budibase/assets/138253412/0641cdc4-8122-45fb-b17d-e12d2cb234cf)

When I tried to set this to number initially when loading the csv file it failed and would only let me load as text.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Thornzee,

This is likely due to the `,` you have after the start of all your numbers. I believe the only symbols you are allowed are `.`, you will need to delete table, fix the commas and then re-import. This should allow you to select number as the type for the column. 

I hope this helps.

**Reply by Thornzee** [reply]
Perfect! This worked. Thank you!

From what I am researching there is no way to format this column as currency, right? It looks like that can only be done on the queries which I don't think is an option if you are using CSV files on the budibase db.

**Reply by ConorWebb96** [reply]
Hi @Thornzee,

There are a few things you can do here; 

- You could use a formula column to add the currency in front of the values. Then use this formula column whenever outputting the values within the design area.
- You could add a symbol in front of these when you are outputting them in the design section e.g. `£  {{ repeater.amount 1 }}`
- you could also do it with queries I guess, but the previous too suggestions are likely easier.

I hope this helps!

**Reply by Thornzee** [reply]
Thanks again. Really helpful. I was able to create a calculated column which now formats the original column as currency. However, I can't figure out how to use that formula column in a form. I tried text, number, and JSON but none worked. Looks like it was discussed here but the solution post has been deleted. [https://github.com/Budibase/budibase/discussions/1033](url)
